pkg/kubectl/resource/builder.go

```diff
@@ -803,3 +803,36 @@ func HasNames(args []string) (bool, error) {
 	}
 	return hasCombinedTypes || len(args) > 1, nil
 }
+
+// HasMultipleTypes returns true if the provided args contain multiple resource kinds
+func HasMultipleTypes(args []string) bool {
+	args = normalizeMultipleResourcesArgs(args)
+	rKinds := sets.NewString()
+	for _, arg := range args {
+		if arg == "all" {
```

Comment:
Unrelated note, we need to think of a better structure for aliases. This and a number places will have issues once we add more aliases in addition to "all" (the only existing one ATM). Might be worth adding an issue.